test.py

```diff
@@ -147,28 +148,33 @@ def send_get_request_to_endpoint(endpoint, path='', base=''):
     msg_headers.pop('omfversion')
     msg_headers["Accept-Verbosity"] = 'verbose'
 
+    # construct and validate url
+    url = urlparse(base+path)
```

Comment:
should there be spaces around the '+'?

Reply:
Not sure, but I suppose it doesn't hurt so I added some